TESTING.md

```diff
@@ -145,44 +145,34 @@ uses the `RemoteLoggingHelper` to create a metric.
 
 ### Testing code that uses Pub/Sub
 
-#### On your machine
-
-You can test against a temporary local Pub/Sub by following these steps:
-
-1. Start the local Pub/Sub emulator before running your tests using `LocalPubSubHelper`'s `create`
-and `start` methods. This will bind a port for communication with the local Pub/Sub emulator.
-  ```java
-  LocalPubSubHelper helper = LocalPubSubHelper.create();
-
-  helper.start(); // Starts the local Pub/Sub emulator in a separate process
-  ```
+You can test against a Pub/Sub emulator:
 
-2. Create and use a `PubSub` object with the options given by the `LocalPubSubHelper` instance. For
-example:
-  ```java
-  PubSub localPubsub = helper.getOptions().getService();
-  ```
-
-3. Run your tests.
+1. [Install Cloud SDK](https://cloud.google.com/sdk/downloads)
 
-4. Stop the local Pub/Sub emulator by calling the `stop()` method, like so:
-  ```java
-  helper.stop();
-  ```
-
-#### On a remote machine
-
-You can test against a remote Pub/Sub emulator as well. To do this, set the `PubSubOptions` project
-endpoint to the hostname of the remote machine, like the example below.
+2. Start the emulator:
+```shell
+$ gcloud beta emulators pubsub start
+```
 
-  ```java
-  PubSubOptions options = PubSubOptions.newBuilder()
-      .setProjectId("my-project-id") // must match project ID specified on remote machine
-      .setHost("<hostname of machine>:<port>")
-      .setCredentials(NoCredentials.getInstance())
-      .build();
-  PubSub localPubsub = options.getService();
-  ```
+3. Point your client to the emulator.
+The code below assumes that you have set the `host` and `port`
+to the host and port the emulator is on.
+```java
+ChannelProvider channelProvider =
+    // SubscriptionAdminSettings works too.
+    TopicAdminSettings.defaultChannelProviderBuilder()
+        .setServiceAddress(host)
+        .setPort(port)
```

Comment:
The emulator doesn't start on any specific port. I just tried starting it twice, the first bound with 8532 and the second with 8261. It *looks* like it just binds to a free port of the kernel's choosing.

After the emulator started, running `gcloud beta emulators pubsub env-init` prints the port the emulator is bound to. I'll just document this.